platform/windows/export/export_plugin.cpp

```diff
@@ -123,6 +123,7 @@ bool EditorExportPlatformWindows::get_export_option_visibility(const String &p_o
 
 void EditorExportPlatformWindows::get_export_options(List<ExportOption> *r_options) {
 	EditorExportPlatformPC::get_export_options(r_options);
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "binary_format/architecture", PROPERTY_HINT_ENUM, "x86_64,x86_32"), "x86_64"));
```

Comment:
Still no arm support possible on Windows?

Reply:
We could add it to the list, but as far as I know we can't build Godot for it yet, so the options won't be helpful.

Reply:
Adding build support should not be too hard to do, but I do not think Vulkan is supported by ARM Windows and OpenGL support is some sort of limited wrapper over DX.